ztp/source-crs/validatorCRs/informDuValidator.yaml

```diff
@@ -27,10 +27,10 @@ status:
         name: load-sctp-module-master
       - apiVersion: machineconfiguration.openshift.io/v1
         kind: MachineConfig
-        name: 02-master-workload-partitioning-master
+        name: 02-master-workload-partitioning
```

Comment:
We should factor this out into a separate PR in case it needs to be backported.

Reply:
Sure. will create a separate PR for this. Removed from this PR.